ui/scss/component/_file-render.scss

```diff
@@ -263,6 +263,8 @@
 .file-render__img-container {
   width: 100%;
   // aspect-ratio: 16 / 9;
+  background-color: #000000;
```

Comment:
also looks like this broke black images https://odysee.com/@SampleDragon:4/ChibiBlepDoggoTransparentFree2Use:a